### DISCUSSION TITLE
Expose Add/Remove connections in the SceneManager.

### DISCUSSION THREAD

**Discussion by FirstGearGames** [FirstGearGames/FishNet#185] [Feature Request]
Requested by Katerlad.

Expose SceneManager.AddConnectionToScene.
Expose SceneManager.RemoveConnectionFromScene.

**Comment by Katerlad** [comment]
To Add to this, this will allow some really cool functionality for clients to load scenes, keep them loaded on the client, and remove them from that scene until later without having to unload and load the scene again. I didnt realize that this was not possible currently for Users to manually pull clients from a scene without unloading.

**Comment by FirstGearGames** [comment]
Added in 3.1.1
